README.md

```diff
@@ -1,6 +1,15 @@
 UI Application for camptocamp.org v6
 ====================================
 
+Requirements
+------------
+
+ * Python 2.7
+ * Node (4.0 or higher)
```

Comment:
I confirm node v4 isn't required, from what I've seen. I only needed v4 to get a slightly more descriptive error message which allowed me to undertand node was attempting to run java behind the scene.